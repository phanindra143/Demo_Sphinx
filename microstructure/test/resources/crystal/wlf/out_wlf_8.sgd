*if,ifprop,eq,2,then
  !c_ij(1,1)=     0.0000
  !c_ij(1,2)=     0.0000
  !c_ij(2,2)=     0.0000
  !c_ij(1,3)=     0.0000
  !c_ij(2,3)=     0.0000
  !c_ij(3,3)=     0.0000
  !c_ij(1,4)=     0.0000
  !c_ij(2,4)=     0.0000
  !c_ij(3,4)=     0.0000
  !c_ij(4,4)=     0.0000
  !c_ij(1,5)=     0.0000
  !c_ij(2,5)=     0.0000
  !c_ij(3,5)=     0.0000
  !c_ij(4,5)=     0.0000
  !c_ij(5,5)=     0.0000
  !c_ij(1,6)=     0.0000
  !c_ij(2,6)=     0.0000
  !c_ij(3,6)=     0.0000
  !c_ij(4,6)=     0.0000
  !c_ij(5,6)=     0.0000
  !c_ij(6,6)=     0.0000
*endif
*if,ifprop,eq,1,then
  !k_ij(1,1)=    85.7300
  !k_ij(1,2)=     1.1821
  !k_ij(2,2)=    88.7505
  !k_ij(1,3)=     0.5808
  !k_ij(2,3)=     0.5622
  !k_ij(3,3)=    85.5991
MP,Kxx,1,   85.72998
MP,Kyy,1,   88.75049
MP,Kzz,1,   85.59905
MP,Kxx,2,   88.75049
MP,Kyy,2,   85.72998
MP,Kzz,2,   85.59905
MP,Kxx,3,   88.75049
MP,Kyy,3,   85.72998
MP,Kzz,3,   85.59905
MP,Kxx,4,   85.72998
MP,Kyy,4,   88.75049
MP,Kzz,4,   85.59905
MP,Kxx,5,   85.72998
MP,Kyy,5,   88.75049
MP,Kzz,5,   85.59905
MP,Kxx,6,   88.75049
MP,Kyy,6,   85.72998
MP,Kzz,6,   85.59905
MP,Kxx,7,   88.75049
MP,Kyy,7,   85.72998
MP,Kzz,7,   85.59905
MP,Kxx,8,   85.72998
MP,Kyy,8,   88.75049
MP,Kzz,8,   85.59905
MP,Kxx,9,   85.59905
MP,Kyy,9,   85.72998
MP,Kzz,9,   88.75049
MP,Kxx,10,   85.72998
MP,Kyy,10,   85.59905
MP,Kzz,10,   88.75049
MP,Kxx,11,   85.72998
MP,Kyy,11,   85.59905
MP,Kzz,11,   88.75049
MP,Kxx,12,   85.59905
MP,Kyy,12,   85.72998
MP,Kzz,12,   88.75049
MP,Kxx,13,   85.59905
MP,Kyy,13,   85.72998
MP,Kzz,13,   88.75049
MP,Kxx,14,   85.72998
MP,Kyy,14,   85.59905
MP,Kzz,14,   88.75049
MP,Kxx,15,   85.72998
MP,Kyy,15,   85.59905
MP,Kzz,15,   88.75049
MP,Kxx,16,   85.59905
MP,Kyy,16,   85.72998
MP,Kzz,16,   88.75049
MP,Kxx,17,   88.75049
MP,Kyy,17,   85.59905
MP,Kzz,17,   85.72998
MP,Kxx,18,   85.59905
MP,Kyy,18,   88.75049
MP,Kzz,18,   85.72998
MP,Kxx,19,   85.59905
MP,Kyy,19,   88.75049
MP,Kzz,19,   85.72998
MP,Kxx,20,   88.75049
MP,Kyy,20,   85.59905
MP,Kzz,20,   85.72998
MP,Kxx,21,   88.75049
MP,Kyy,21,   85.59905
MP,Kzz,21,   85.72998
MP,Kxx,22,   85.59905
MP,Kyy,22,   88.75049
MP,Kzz,22,   85.72998
MP,Kxx,23,   85.59905
MP,Kyy,23,   88.75049
MP,Kzz,23,   85.72998
MP,Kxx,24,   88.75049
MP,Kyy,24,   85.59905
MP,Kzz,24,   85.72998
*endif
