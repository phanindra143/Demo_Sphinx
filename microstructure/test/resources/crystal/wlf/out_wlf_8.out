            ifx                 q_x                 q_y                 q_z     
  /% source: R:/166_320/Ablage_Simulationen/Modellbildung_Simulation_CMC-SiC_[Vorlauf+WiMi+MaVo]/04_04_2017_Modellierung_Matrix_mikro_ditt_[WiMi]/2_Waermeleitfähigket_SiC-Si\WLF-50SiC-Poren\12
             3.00000      -1338960.73243        -20722.74510         33962.28196
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             3.00000         15625.00000           -68.43526          -614.09651
  /%
            ifx                 q_x                 q_y                 q_z     
  /%
             2.00000        -23731.47126      -1386997.60520         -8806.90989
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             2.00000           162.46295         15625.00000           -61.07932
  /%
            ifx                 q_x                 q_y                 q_z     
  /%
             1.00000         31151.53946        -25498.58871      -1337811.07924
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             1.00000          -360.07426           246.98629         15625.00000
  /%
